Source/WebCore/html/CanvasBase.h

```diff
@@ -149,15 +150,14 @@ class CanvasBase {
 
 private:
     bool shouldInjectNoiseBeforeReadback() const;
-    void postProcessDirtyCanvasBuffer() const;
     virtual void createImageBuffer() const { }
 
     mutable IntSize m_size;
-    mutable FloatRect m_postProcessDirtyRect;
     mutable Lock m_imageBufferAssignmentLock;
     mutable RefPtr<ImageBuffer> m_imageBuffer;
     mutable size_t m_imageBufferCost { 0 };
     mutable std::unique_ptr<GraphicsContextStateSaver> m_contextStateSaver;
+    mutable CanvasNoiseInjection m_canvasNoiseInjection;
```

Comment:
It seems the CanvasNoiseInjection doesn't need to be mutable, as postprocessdirty is only used in mutable scope.